helper_lib.sh

```diff
@@ -77,7 +77,16 @@ get_docker_cumulative_command_line_args() {
 # Does not account for option default or implicit options.
 get_docker_effective_command_line_args() {
     OPTION="$1"
-    get_docker_cumulative_command_line_args $OPTION | tail -n1
+    get_docker_cumulative_command_line_args "$OPTION" | tail -n1
+}
+
+get_docker_configuration_file_args() {
+    OPTION="$1"
+    FILE="$(get_docker_effective_command_line_args '--config-file' | \
```

Comment:
Updated so it falls back to default if a `--config-file` file isn't there, https://github.com/docker/docker-bench-security/pull/218/commits/1f499387c6c687a631b6050258412a89019565f4